PWGCF/Tasks/correlations.cxx

```diff
@@ -724,8 +741,11 @@ struct CorrelationTask {
       case 2212: // proton
       case -2212:
         return 2;
-      default:
+      case 421: // D0
+      case -421:
         return 3;
+      default:
+        return 4;
```

Comment:
I would suggest using named constants from `TPDGCode.h` extended by Common/Constants/include/CommonConstants/PhysicsConstants.h if needed

Reply:
Yes, this is something we should probably change eventually.